lib/services/cordova-plugins.ts

```diff
@@ -103,7 +104,9 @@ export class CordovaPluginsService implements ICordovaPluginsService {
 
 	public configure(): void {
 		let future = new Future();
-		let params = ["set", "registry", this.$config.CORDOVA_PLUGINS_REGISTRY];
+
+		// Need to set plugman's registry to use npm registry to avoid problems with cordova registry availability.
+		let params = ["set", "registry", "http://registry.npmjs.org"];
```

Comment:
Can't we just change the value of the variable, I believe it's written in config-base.json